src/Context/Annotation/ContextDefinition.php

```diff
@@ -33,6 +34,13 @@ class ContextDefinition extends CoreContextDefinition {
    * {@inheritdoc}
    */
   public function __construct(array $values) {
+    // Filter out any @Translation annotation objects.
+    $old_values = $values;
```

Comment:
I'm a little reluctant to write into a PHP array that's used in a foreach() loop, lest I cause something to reset :-)  If that's safe, then I can remove that.